frontend/src/host_orchestrator/orchestrator/instancemanager_test.go

```diff
@@ -42,6 +42,7 @@ func (AlwaysSucceedsValidator) Validate() error {
 }
 
 const fakeLatesGreenBuildID = "9551522"
+const cvdUser = "_cvd-executor"
```

Comment:
nit: wondering whether having the same production user name here could cause confusion in the future while reading test failure messages.

Reply:
Changed